spec/order_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'order'
+
+describe Order do
+  subject(:order) { described_class.new}
+  let(:subject) {double(order)}
+
+  describe 'Make an order' do
+    it 'creates menu' do
+      expect(order.current_order).to include {}
+    end
+
+    it 'selects item' do
+      order.select_item(:chips, 2)
+      expect(order.current_order).to include :chips
+    end
+
+    it 'raises error when item not on menu selected' do
+      expect {order.select_item('dogshit', 2)}.to raise_error 'please select item from menu'
```

Comment:
Metrics/LineLength: Line is too long. [92/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)